coalib/bearlib/abstractions/Lint.py

```diff
@@ -213,6 +214,8 @@ def _check_command(cls, executable, command, fail_msg):
         if cls._check_executable(executable):
             if command is None:
                 return True  # when there are no prerequisites
+            if isinstance(command, str):
```

Comment:
Can we not do this ? 
I think it makes sense to just restrict this to a list.
`shlex` has some functions that just don't work well with windows - and im not sure if `.split()` wokrs find on windows ...